packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -58,6 +65,13 @@ export interface ToolbarItemProps
    * @param {object} data - All props.
    */
   onBlur?: ComponentEventHandler<ToolbarItemProps>
+
+  /**
+   * Attaches a `Popup` component to the ToolbarItem.
+   * Accepts all props as a `Popup`, except `trigger` and `children`.
+   * Sets `accessibility` to `popupFocusTrapBehavior` by default.
+   */
+  popup?: Omit<PopupProps, 'trigger' | 'children'>
```

Comment:
```suggestion
  popup?: Omit<PopupProps, 'trigger' | 'children'> | string
```
String can still be valid type here, we can map it to the content of the Popup.